eve/main.yml

```
@@ -2460,7 +2389,9 @@ stages:
             FILE_SOURCE: >-
               example-solution-%(prop:example_solution_version)s.iso
             FILE_DEST: example-solution.iso
-            BASE_URL: "%(prop:artifacts_private_url)s"
+            BASE_URL: "http://artifacts/builds/github: \
+                       scality:metalk8s-solution-example: \
+                       promoted-%(prop:example_solution_version)s"
```

Won't this resolve to `http://artifacts/builds/github: scality:metalk8s-solution-example: promoted-%(prop:example_solution_version)s` (with spaces), and cause problems down the line?

```suggestion
            BASE_URL: "http://artifacts/builds/github:\
                       scality:metalk8s-solution-example:\
                       promoted-%(prop:example_solution_version)s"
```

---

Yes, totally.